script.js

```diff
@@ -0,0 +1,197 @@
+  document.querySelector(".todo-input").value = "";
+
+  // todo item 개수 갱신
+  countTodoItem();
+
+  // localStorage에 갱신된 todoList 저장
+  saveTodoLocalStorage();
+};
+
+// todo item을 삭제
+const deleteTodoItem = (e) => {
+  const target = e.target.parentNode;
+  document.querySelector(".todo-list").removeChild(target);
+
+  // target node 삭제
+  const newTodoList = todoList.filter(
+    (item) => item.id !== parseInt(target.id),
+  );
+  todoList = newTodoList;
+
+  // todo item 개수 갱신
+  countTodoItem();
+
+  // localStorage에 갱신된 todoList 저장
+  saveTodoLocalStorage();
+};
+
+// todo item 개수 출력
+const countTodoItem = () => {
+  const count = document.querySelector(".todo-count");
+  count.innerText = "(" + todoList.length + ")";
+};
+
+// todo item -> done item
+const toggleTodoToDone = (e) => {
+  // todo List에서 item 삭제
+  deleteTodoItem(e);
+
+  // done list에 item 추가
+  paintDoneItem(e.target.innerText);
+};
+
+// 화면에 todo done item을 추가
+const paintDoneItem = (text) => {
+  const todoId = new Date().valueOf();
+  const newTodoDone = document.createElement("li");
+  const newTodoText = document.createElement("span");
+  const todoDel = document.createElement("img");
+
+  // todo done item 텍스트 추가
+  newTodoText.innerText = text;
+  newTodoText.style.cursor = "pointer";
+  newTodoText.style.fontSize = "15px";
+  newTodoText.style.textDecorationLine = "line-through";
+  newTodoText.style.color = "lightGrey";
+  newTodoText.style.paddingLeft = "5px";
+  newTodoText.addEventListener("dblclick", toggleDoneToTodo);
+
+  // todo done item 삭제 버튼 추가
+  todoDel.setAttribute("src", "img/bin.png");
+  todoDel.style.width = "14px";
+  todoDel.style.paddingLeft = "5px";
+  todoDel.style.cursor = "pointer";
+  todoDel.addEventListener("click", deleteDoneItem);
+
+  // li에 item 추가
+  newTodoDone.setAttribute("id", todoId);
+  newTodoDone.style.marginBottom = "13px";
+  newTodoDone.appendChild(newTodoText);
+  newTodoDone.appendChild(todoDel);
+
+  const todoObj = {
+    id: todoId,
+    text,
+  };
+
+  doneList.push(todoObj);
+  document.querySelector(".done-list").appendChild(newTodoDone);
+
+  // done item 개수 갱신
+  countDoneItem();
+
+  // localStorage에 갱신된 doneList 저장
+  saveDoneLocalStorage();
+};
+
+// done item -> todo item
+const toggleDoneToTodo = (e) => {
+  // done List에서 item 삭제
+  deleteDoneItem(e);
+
+  // todo list에 item 추가
+  paintTodoItem(e.target.innerText);
+};
+
+// todo done item을 삭제
+const deleteDoneItem = (e) => {
+  const target = e.target.parentNode;
+  document.querySelector(".done-list").removeChild(target);
+
+  // target node 삭제
+  const newTodoList = doneList.filter(
+    (item) => item.id !== parseInt(target.id),
+  );
+  doneList = newTodoList;
+
+  // done item 개수 갱신
+  countDoneItem();
+
+  // localStorage에 갱신된 doneList 저장
+  saveDoneLocalStorage();
+};
+
+// todo item 개수 출력
+const countDoneItem = () => {
+  const count = document.querySelector(".done-count");
+  count.innerText = "(" + doneList.length + ")";
```

Comment:
이 부분도 템플릿 리터럴을 사용해 보시면 좋을 것 같아요

```suggestion
  count.innerText = `(${doneList.length})`;
```

Reply:
넵 앞으로 템플릿 리터럴 사용하도록 해야겠어요!!